### DISCUSSION TITLE
Invalid chainId 1.

### DISCUSSION THREAD

**Discussion by deeplearning335** [WalletConnect/walletconnect-monorepo#3640] [Developer Support] [answered]
I am going to migrage my dapp to walletconnect v2.0.
I have seen the document. 
But the [website](https://web3-react-mu.vercel.app/) is not working.
I am facing the error.

**Error: Invalid chainId 1. Make sure default chain is included in "chains" - chains specified in "optionalChains" may not be selected as the default, as they may not be supported by the wallet.**
What is wrong?
How can I fix this error?
Help me.

**Comment by glitch-txs** [comment, marked as answer]
hey that's a web3-react implementation, I think you should open an issue there
